client/lib/application/src/application-lifecycle/components/Charts.jsx

```diff
@@ -20,7 +20,7 @@ const Charts = (props) => {
                     <Link
                         to={Routes.verApproval({applicationId: props.applicationId, versionId: version.id})}
                         className='btn btn-default btn-small'>
-                        <Icon name='check' />
+                        <Icon fixedWidth name='check' />
                     </Link>
                     Approvals
```

Comment:
I think `Approvals` has to be on the same line, otherwise you miss a space between the button and the text? Like:

~~~ jsx
</Link> Approvals
~~~